### ISSUE TITLE
Meizu 16th cannot use version 2.1 SCRCPy.

### ISSUE THREAD

**Issue by zzh223** [Genymobile/scrcpy#4151]
Version 2.0 can be used normally, but when I upgraded to version 2.1, I could not load the screen of the mobile phone, please fix it, thank you!

**Comment by rom1v** [comment]
Thank you for your report.

Duplicate of #4143 #4146.

**Comment by zzh223** [comment]
> Thank you for your report.
> 
> Duplicate of #4143 #4146.

Thank you for your reply. Can you fix the problem of Meizu's mobile phone? I have a number of different brands of mobile phones, Scrcpy2.1 version support vivo mobile phone forwarding audio, great.

**Comment by rom1v** [comment]
I'd like to fix it, but I don't have Meizu phones, so I need help to find the culprit commit between v2.0 and v2.1: https://github.com/Genymobile/scrcpy/issues/4143#issuecomment-1625309668

**Comment by zzh223** [comment]
> 我想修复它，但我没有魅族手机，所以我需要帮助来找到 v2.0 和 v2.1 之间的罪魁祸首： [#4143 （评论）](https://github.com/Genymobile/scrcpy/issues/4143#issuecomment-1625309668)

It doesn't prompt any error, just no window display after startup, which is very strange.

**Comment by rom1v** [comment]
Yes, I know, someone with a Meizu phone must build from sources and git bisect to find from which commit this behavior happens (cf linked comment).

**Comment by zzh223** [comment]
> Yes, I know, someone with a Meizu phone must build from sources and git bisect to find from which commit this behavior happens (cf linked comment).

@rom1v  I tested the patch (https://github.com/Genymobile/scrcpy/issues/4143#issuecomment-1627980963), it can work properly on my meizu phone, other several different brands of mobile phone can normal work.

**Comment by Bertaz** [comment]
Fixed by 7e936fa879d9ee37608d9d0ce36f66ea06a6317d, confirmed by author

> I tested the patch [#4143 (comment)](https://github.com/Genymobile/scrcpy/issues/4143#issuecomment-1627980963), it can work properly on my meizu phone, other several different brands of mobile phone can normal work.
